kernelci/api/models.py

```diff
@@ -203,6 +203,9 @@ class Node(DatabaseModel):
     jobfilter: Optional[List[str]] = Field(
         description="Restrict jobs that can be scheduled by this node"
     )
+    platformfilter: Optional[List[str]] = Field(
```

Comment:
I'd suggest using `_` between words for better readability as per python naming conventions.
`platform_filter`.